### DISCUSSION TITLE
Hot Loading?

### DISCUSSION THREAD

**Discussion by Timoti** [responsively-org/responsively-app#1249] [Q&A]
Somewhere on the website  I saw mention of "developer hot reloading"?
I would dearly dearly love to be able to have automatic reloading  on save - especially css / js only reloads.
ATM, I've hacked something using Keyboard Maestro so I can reload while I'm in another app, but even this won't only reload CSS, which is what I really need.
If extensions could be installed, I'd install a CSS refresh extension, but obviously that's not an option either.

Is Hot reloading  / css/js only reloading a thing?  (sorry cant find wherever i saw the mention) 

Can it be?

**Comment by minowau** [comment]
I Have developed the hot reloading to the software you can check my request #1265
